### PR TITLE
Import data command

### LINKED ISSUE #13: Create management command to import data
Create management command to import data from #8

### PR DESCRIPTION
**Pull request by froi**
A custom manage command to import scrapped data from suministrospr.com

Closes #13

### CODE REVIEW

**Comment by froi** [comment]
Working on `municipality` name normalization.

**Comment by froi** [comment]
Data after import

image

**Comment by froi** [comment]
Sample command execution

image

File: suministrospr/suministros/management/commands/import_data.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                    data = json.load(json_file)
+                    for item in data:
+                        suministro = Suministro()
+                        suministro.content = item['content']
+                        suministro.title = item['title']
+
+                        municipio = self.normalize(item.get('municipio', ''))
+                        
+                        if municipio in MUNICIPALITIES:
+                            suministro.municipality = item['municipio']
```
Comment (jpadilla) [inline suministrospr/suministros/management/commands/import_data.py]:
```suggestion
                            suministro.municipality = municipio
```

File: suministrospr/suministros/management/commands/import_data.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
+from django.core.management.base import BaseCommand, CommandError
+from ...models import Suministro
+from os import path
+import glob
+import json
+from ...constants import MUNICIPALITIES
+import unicodedata
```
Comment (jpadilla) [inline suministrospr/suministros/management/commands/import_data.py:5]: nitpick (import order):

```python
import glob
import json
from os import path

import unicodedata
from django.core.management.base import BaseCommand, CommandError

from ...models import Suministro
from ...constants import MUNICIPALITIES
```

**Review by jpadilla — APPROVED** [review]

File: suministrospr/suministros/management/commands/import_data.py (hunk 2 above)
Comment (froi) [reply suministrospr/suministros/management/commands/import_data.py:5]: Done.

**Pull request merged**